app/assets/stylesheets/foundation_and_overrides.scss

```diff
@@ -1441,3 +1441,32 @@ $include-html-global-classes: $include-html-classes;
 // $include-accessibility-classes: true;
 
 @import 'foundation';
+
+.dash-header {
+  background-color: $topbar-bg-color;
+}
+
+.footer {
+  background-color: $primary-color;
+  text-align: center;
+  color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.